packages/filebrowser/style/base.css

```
@@ -278,6 +288,11 @@
   text-align: right;
 }
 
+.jp-DirListing-itemFileSize {
+  flex: 0 0 84px;
```

```suggestion
  flex: 0 0 90px;
```

(needed when increasing `.jp-DirListing-headerItem.jp-id-filesize`)